docs/tutorial/1-serialization.md

```diff
@@ -235,15 +235,15 @@ The root of our API is going to be a view that supports listing all the existing
         if request.method == 'GET':
             snippets = Snippet.objects.all()
             serializer = SnippetSerializer(snippets, many=True)
-            return JsonResponse(serializer.data)
+            return JsonResponse(serializer.data, safe=False)
```

Comment:
Let's *only* use this for the list response (here), since that's the only place it's needed.
We should be able to drop `safe=False` in the other lines.